src/scss/_main.scss

```diff
@@ -1138,7 +1138,7 @@ details.build-toggle {
   scrollbar-width: thin;
 }
 
-.logs-table tr td:nth-child(2) {
+.logs-table .line-content {
```

Comment:
too generic, it started targeting children unintentionally when adding annotations